src/components/table/OcTable.vue

```diff
@@ -216,8 +399,89 @@ export default {
     footerColspan() {
       return this.fields.length
     },
+    //my computed
+    groupedItems() {
+      let result = this.groupingFunction(
+        this.selected,
+        this.tableData,
+        this.groupingOrderAsc[this.selected]
+      )
+
+      if (this.groupingSettings && this.groupingSettings.previewTable && this.previewEnabled) {
+        let previewCount = this.groupingSettings.previewAmount + 1
+
+        result.forEach(e => {
+          let eLength = e.data.length
+          if (eLength - previewCount >= 0) e.data = e.data.slice(0, previewCount - 1)
+          previewCount = previewCount - eLength
+        })
+
+        result = result.filter(e => e.data.length > 0)
+      }
+
+      this.resultArray = result
```

Comment:
Am I understanding correctly that you store this in the state to allow you to hide/show the rows, right?
This seems a bit messy, but I don't know enough vue to know if it's necessary or not. I guess I'll let oc review this after.